airgun/views/smart_variable.py

```diff
@@ -0,0 +1,149 @@
+from widgetastic.widget import (
+    Checkbox,
+    ParametrizedLocator,
+    Select,
+    Text,
+    TextInput,
+    View,
+)
+from widgetastic_patternfly import BreadCrumb
+
+from airgun.views.common import BaseLoggedInView, SearchableViewMixin
+from airgun.widgets import FilteredDropdown, SatTable
+
+
+class MatcherAttribute(View):
+    """Represent smart variable matcher attribute pair. Usually, it looks like
+    as two fields separated by '=' mark
+    """
+    matcher_attribute_type = Select(
+        ".//select[contains(@class, 'matcher_key')]")
+    matcher_attribute_value = TextInput(
+        locator=".//input[contains(@class, 'matcher_value')]")
+
+
+class SmartVariableContent(View):
+    ROOT = ParametrizedLocator('{@locator}')
+    key = TextInput(locator=".//input[contains(@name, '[key]')]")
+    description = TextInput(
+        locator=".//textarea[contains(@name, '[description]')]")
+    puppet_class = FilteredDropdown(id='variable_lookup_key_puppetclass_id')
+    key_type = Select(locator=".//select[contains(@name, '[key_type]')]")
+    default_value = TextInput(
+        locator=".//textarea[contains(@name, '[default_value]')]")
+    hidden = Checkbox(locator=".//input[contains(@name, '[hidden_value]')]")
+
+    def __init__(self, parent, locator, logger=None):
+        View.__init__(self, parent, logger=logger)
+        self.locator = locator
+
+    @View.nested
+    class optional_input_validators(View):
+        expander = Text(
+            ".//h2[contains(@data-target, '#optional_input_validators_')]")
+        validator_type = Select(
+            locator=".//select[contains(@name, '[validator_type]')]")
+        validator_rule = TextInput(
+            locator=".//input[contains(@name, '[validator_rule]')]")
+
+        def __init__(self, parent, logger=None):
+            View.__init__(self, parent, logger=logger)
+            if 'collapsed' in self.browser.element(
```

Comment:
why not `self.browser.classes`?

Reply:
hmm, as far as I tried and according to documentation - it will return set of classes, but I need simple string

Reply:
and what's wrong with set (or list)? you'll have the same check - `if 'collapsed' in self.browser.classes`

Reply:
ok, let's me test